### PR TITLE
Features/add time zone translation

### PR DESCRIPTION
**Pull request by navidehsa**

### CODE REVIEW

File: src/components/MaterialUI/Inputs/DatePicker.js [hunk 1]
```diff
@@ -116,7 +117,7 @@ const WrappedDatePicker = ({
 						onChange={date => updateDate(date, metadata)}
 						showTimeInput={useTime ?? false}
 						useTime={useTime ?? false}
-						customTimeInput={useTime ? <TimePicker showTimeZone={showTimeZone} /> : null}
+						customTimeInput={useTime ? <TimePicker showTimeZone={showTimeZone} timeZone={timePickerTimeZone} /> : null}
```
Comment (Anstsiya) [inline src/components/MaterialUI/Inputs/DatePicker.js:120]: don't you think we can pass here already translated timezone somehow? Cause we have all this timezones and translations in OMS for e.g. So maybe it's better than to have sooooo huge translations files with all timezones that some we even will never ever use

Comment (Anstsiya) [reply src/components/MaterialUI/Inputs/DatePicker.js:120]: actually getting timezones requests and selectors it's even a part of orc-shared. Can't we use just selector to get translation by timezone id ?

Comment (Anstsiya) [reply src/components/MaterialUI/Inputs/DatePicker.js:120]: I checked and it's possible to get timezones inside the timepicker.
I tried just to get all timezones like this 
image
and it works. So there should be no problems to pass timezone id and use localizedTimezone selector

Comment (navidehsa) [reply src/components/MaterialUI/Inputs/DatePicker.js:120]: @Anstsiya as far as I know the API does not return the translations and just return the English name,let me know if I am wrong!

**Review by Anstsiya — DISMISSED** [review]

**Review by Anstsiya — APPROVED** [review]

**Review by Anstsiya — APPROVED** [review]

**Pull request merged**